Sources/SwiftSDKGenerator/SwiftSDKRecipes/LinuxRecipe.swift

```diff
@@ -21,9 +21,10 @@ public struct LinuxRecipe: SwiftSDKRecipe {
     case remoteTarball
   }
 
-  public enum HostSwiftSource: Sendable {
+  public enum HostSwiftSource: Sendable, Equatable {
     case localPackage(FilePath)
     case remoteTarball
+    case skip
```

Comment:
IMO `skip` is more appropriate for an action case. If the name of the type was `HostSwiftGeneratorAction`, then you'd allow the action to skip this step. But with the current name something like `preinstalled` feels more appropriate?

```suggestion
    case preinstalled
```

Reply:
Okay that does make sense! `preinstalled` sounds good to me!